Makefile

```diff
@@ -52,7 +52,7 @@ itest: require_os package
 		-v `pwd`/dist:/dist:ro \
 		docker-dev.yelpcorp.com/$(OS)_yelp \
 		/itest/$(OS).sh \
-		/dist/$(OS)/$(PACKAGE_NAME)_$(VERSION)+yelp-$(BUILD_NUMBER)_amd64.deb
```

Comment:
The version change here (I think something in the `fpm-cookery` version change led to this) means that new versions such as `4.5.3-2677` will actually compare to be lower than the existing versions such as `4.5.3+yelp-2288`, so that's not really workable and is probably something I need to fix before merging this.

Reply:
Yeah, this appears to be mainly due to the change in https://github.com/bernd/fpm-cookery/commit/e68229a371b119043214d5f609ab27d8d19ce4b8 that changed versioning from `<version>+<vendor><revision>` to `<version>-<revision>+<vendor>`, so this would move from `4.5.3+yelp-2288` to `4.5.3-2288+yelp` for instance. That doesn't order properly and the revision separator isn't customizable, so I've gone with putting it all into `revision` as the build number plus `-yelp`, and `dpkg --compare-versions "4.5.3+yelp-2653" "lt" "4.5.3-2653-yelp"` confirms that it does order properly. I've pushed a new commit to address the versioning issue.

This will also need a change to be made to how the version is defined in the puppet repo though, but that doesn't look too bad.